packages/makeswift/src/integrate-next-app.ts

```diff
@@ -5,8 +5,40 @@ import path from 'path'
 import { manipulateNextConfig } from './utils/manipulate-next-config'
 import { yarnOrNpm } from './utils/yarn-or-npm'
 import { createFolderIfNotExists } from './utils/create-folder-if-not-exists'
+import inquirer from 'inquirer'
+import chalk from 'chalk'
+
+async function getApprovalToIntegrate(dir: string): Promise<boolean> {
+  const projectName = dir.split('/').reduce((prev, curr) => curr)
```

Comment:
`reduce` will throw an error if the array is empty and an initial value is not given. Should we pass an initial value? Or will this never be of length 0? Also, is this just trying to get the last item in the array? Seems convoluted instead of doing something like this:

```suggestion
  const projectName = dir.split('/').slice(-1)[0]
```

or

```suggestion
  const projectName = dir.split('/').at(-1)
```